docker/js_ci/run.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env sh
+
+set -o errexit
+set -o nounset
+set -o verbose
+
+find /data/*.js* | xargs jshint
```

Comment:
I'd rather try to extract the "run" step into a separate bash script for consistency (some builds will require multiple commands), and it's generally less fiddly to run shell commands in a shell.